src/main/java/quinn/task/Event.java

```diff
@@ -0,0 +1,25 @@
+package quinn.task;
+
+public class Event extends Task {
+    private final String fromDateTimeInput;
+    private final String toDateTimeInput;
```

Comment:
Perhaps using startDateTime and endDateTime is more descriptive and concise here. Furthermore adding the word 'input' in the naming dosent't add significant value and can make the code more verbose.